aspnetcore/blazor/fundamentals/signalr.md

```diff
@@ -1511,6 +1511,72 @@ app.MapBlazorHub();
 
 [!INCLUDE[](~/blazor/security/includes/httpcontext.md)]
 
+## Impersonation for Windows Authentication
+
+Authenticated hub connections (<xref:Microsoft.AspNetCore.SignalR.Client.HubConnection>) are created with <xref:Microsoft.AspNetCore.Http.Connections.Client.HttpConnectionOptions.UseDefaultCredentials%2A> to indicate the use of default credentials for HTTP requests:
+
+```csharp
+protected override async Task OnInitializedAsync()
```

Comment:
Not sure we should show this.

Reply:
I think you mean drop the entire method ... "this" is hard to interpret 😆.

I'll give it 🔪 the chop.